src/nupic/algorithms/Connections.cpp

```diff
@@ -40,11 +40,11 @@ using namespace nupic;
 using namespace nupic::algorithms::connections;
 using nupic::sdr::SDR;
 
-Connections::Connections(CellIdx numCells, Permanence connectedThreshold) {
-  initialize(numCells, connectedThreshold);
+Connections::Connections(CellIdx numCells, Permanence connectedThreshold, bool timeseries) {
```

Comment:
maybe instead of timeseries, some `ignoreRepetitiveUpdates`,...?

Reply:
I like the work `timeseries` because it's descriptive of when to use this feature.